.github/workflows/publish.yml

```diff
@@ -29,8 +27,8 @@ jobs:
               git config pull.rebase false &&
               git config user.email "bot@lunie.io" &&
               git config user.name "Publish Lunie Bot" &&
-              git remote add bot https://${GIT_BOT_TOKEN}@github.com/luniehq/lunie.git &&
-              git checkout master &&
+              git remote set-url origin https://${GIT_BOT_TOKEN}@github.com/luniehq/lunie.git &&
+              git checkout fabo/merge-test &&
```

Comment:
Shouldn't it be master here?

Reply:
OK, I see it is good on develop. Nice

Reply:
good catch. I fixed it right away on develop